Deprecate plural outputs, unify inputs and outputs (#210)
metatensor/metatomic: docs/src/quantities/index.rst

.. _standard-quantities:

Standard quantities
===================

In order for multiple simulation engines to be able use arbitrary metatomic
models to compute atomic properties, we need all the models to use the same
metadata when handling the same quantity. If your model returns one of the
quantity defined in this documentation as output; or use them as input; then it
must follow the metadata structure described here.

If you need other quantities as inputs or outputs, you should use custom
quantity with a name containing ``::``, such as ``my_code::my_quantity``. For
such custom quantity, you are free to use any relevant metadata structure, but
if multiple people are using the same quantity, they are encouraged to come
together, define the metadata schema they need and add a new section to these

> pages.

Reviewer: Should we mention here that they are usually in singular...

Author: I don't think we have to, we can discuss this whenever a PR comes in. If this becomes an issue we can start having docs for how to add new quantities

.. toctree::
  :maxdepth: 1
  :hidden:

  energy
  non_conservative
  mass
  position
  momentum
  velocity
  charge
  heat_flux
  spin_multiplicity
  feature

>   variants

Reviewer: ```suggestion
  variant
```

?

Author: no, there are multiple variants of the same quantity

Variants
^^^^^^^^

> Models can define variants of any quantity, for example to provide the same

Reviewer: ```suggestion
Models can define a variant of any quantity, for example to provide the same
```

      when running multiple models simultaneously.

    .. grid-item-card:: Energy uncertainty
      :link: energy-uncertainty-quantity
      :link-type: ref

      .. image:: /../static/images/energy-uncertainty-quantity.png

      The uncertainty on the potential energies, useful to quantify the confidence of
      the model.

    .. grid-item-card:: Non-conservative force
      :link: non-conservative-force-quantity
      :link-type: ref

      .. image:: /../static/images/nc-force-quantity.png

      Forces directly predicted by the model, not derived from the potential
      energy.

    .. grid-item-card:: Non-conservative stress
      :link: non-conservative-stress-quantity
      :link-type: ref

      .. image:: /../static/images/nc-stress-quantity.png

      Stress directly predicted by the model, not derived from the potential
      energy.

    .. grid-item-card:: Mass
      :link: mass-quantity
      :link-type: ref

      .. image:: /../static/images/mass-quantity.png

      Atomic masses

    .. grid-item-card:: Position
      :link: position-quantity
      :link-type: ref

      .. image:: /../static/images/position-quantity.png

>       Atomic positions predicted by the model, to be used in ML-driven simulations.

Reviewer: To be in line with the nc descriptions.

```suggestion
      Atomic positions directly predicted by the model
```
